brave-unbreak.txt

```diff
@@ -1,3 +1,4 @@
+[Adblock Plus 2.0; uBlock Origin]
```

Comment:
do we need this on the entries under `brave-lists/*` as well?

Reply:
@antonok-edm Actually, the `.gitattributes` file would be enough for this, since it overrides the language for the given patterns, and this behavior is useful for handling sections, eg. `brave-lists/*.txt` in this repo. However, it is advisable to specify the adblock agent for complete lists. It is important to know that one of the conditions of Linguist support is the fulfillment of the [**popularity threshold**](https://github.com/github/linguist/blob/master/CONTRIBUTING.md#adding-an-extension-to-a-language), and Linguist only examines the adblock agent with a heuristic. If the use of the adblock agent is reduced (because it is replaced by a gitattributes file), then the heuristics support may cease in the future. However, this heuristic is the only way to clearly distinguish a legit txt file from an adblock list, so the agent is one of the cornerstones of adblock language support.

Therefore, if possible, use the agent for the complete list, but you can use .gitattributes for partial lists (or call it sections or something). Anyway, you can also use the agent for sections, that's also good.

Reply:
@scripthunter7 makes sense, thanks for the detailed answer!